Description: Require directx_shaders.c only on Windows
 This file is only needed on Windows,
 so don't complain if it's not present
 on other platforms.
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/addons/primitives/CMakeLists.txt
+++ b/addons/primitives/CMakeLists.txt
@@ -6,7 +6,6 @@
     prim_soft.c
     point_soft.c
     primitives.c
-    directx_shaders.c
     )
 
 if(WIN32)
